backend/src/main/java/de/neuefische/backend/model/Wizard.java

```diff
@@ -1,10 +1,12 @@
 package de.neuefische.backend.model;
 import lombok.With;
+import org.springframework.data.annotation.Id;
 
 import java.util.Objects;
 
 @With
```

Comment:
Add @ Document Annotation for the collection name

Reply:
ok we do this one little ding